docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -0,0 +1,167 @@
+EOL for major versions of Aiven Services
+========================================
+
+End of Life (EOL) refers to the deadline after which affected Aiven
+services are retired and will be no longer supported nor maintained.
+
+Since August 2020, Aiven aims to follow the EOL schedule set by the
+original authors and maintainers of the open source software, aka
+upstream projects. Once the upstream project retires a specific version,
+they do not receive security updates and critical bug fixes anymore by
+the maintainers.
+
+Continued use of outdated services means that they no longer offer our
+customers the level of protection their business needs. Therefore, by
+following the upstream project's EOL schedule, we ensure that Aiven
+services are always running on supported versions of the open source
+software.
+
+**Version numbering**
+~~~~~~~~~~~~~~~~~~~~~
+
+Aiven services inherit the upstream project’s software versioning
+scheme. Depending on the service, a major version can be either a single
+digit (e.g. PostgreSQL® 12) or ``major.minor`` (e.g. Kafka® 2.4). The
+exact version of the service is visible in the Aiven console once the
+service is up and running.
+
+Aiven for Elasticsearch
```

Comment:
Is this already EOL? Is it still relevant?

Reply:
Phase 2 of the EOL for Elasticsearch is currently under progress where we remove the backwards compatibility support for Elasticsearch in OpenSearch. 

We'll need to update this section to indicate that we are offering Aiven for OpenSearch as a replacement product, update the section heading to Aiven for OpenSearch and add new entries for OpenSearch versions (we are currently preparing to release v2).

Reply:
@sbijjal I've added the OpenSearch section I only managed to find the EOL for upstream v1 is `2023-12-31`
Could you provide the EOL for Aiven and upstream? thank you.